### PR TITLE
Grafana plugin integration

### PR DESCRIPTION
**Pull request by SANTOSH-MWP**
[https://mavenwave.atlassian.net/browse/MC-147](url)

Included-
   Grafana plugins
   configurations
   docker-compose in general for further utilization

### CODE REVIEW

File: app-config.yaml [hunk 1]
```diff
@@ -48,37 +47,45 @@ backend:
...
 
 proxy:
-  '/test':
-    target: 'https://example.com'
+  "/test":
+    target: "https://example.com"
     changeorigin: true
+
+  "/grafana/api":
+    target: https://backstage.devstats.cncf.io
```
Comment (fosterm-mw) [inline app-config.yaml:82]: Why are we pointing to devstats.cncf's backstage instance here?

File: app-config.yaml [hunk 2]
```diff
@@ -48,37 +47,45 @@ backend:
...
+    headers:
+      Authorization: Bearer ${GRAFANA_TOKEN}
+
+grafana:
+  domain: https://backstage.devstats.cncf.io
```
Comment (fosterm-mw) [inline app-config.yaml:87]: Again, why are we pointing to devstats cncf's backstage?

File: catalog-info.yaml [hunk 3]
```diff
@@ -37,6 +37,16 @@ apiVersion: backstage.io/v1alpha1
...
+  annotations:
+    grafana/tag-selector: "Kubernetes"
+  tags:
+    - web
+    - frontend
+    - backstage
+  links:
+    - url: https://backstage.xxxxxx.xxx.xx
+      title: Backstage
+      icon: cloud
```
Comment (fosterm-mw) [inline catalog-info.yaml]: Add this to a different resource. I was using this resource for gcp-folder.

**Review by fosterm-mw — CHANGES_REQUESTED** [review]

File: app-config.yaml (hunk 1 above)
Comment (SANTOSH-MWP) [reply app-config.yaml:82]: just used as an instance open path to fulfill the request, we will replace with our own hosting.

File: app-config.yaml (hunk 2 above)
Comment (SANTOSH-MWP) [reply app-config.yaml:87]: just used as an instance open path to fulfill the request, we will replace with our own hosting

File: catalog-info.yaml (hunk 3 above)
Comment (SANTOSH-MWP) [reply catalog-info.yaml]: entry removed from catalog-info.
as of now configuration loading from entity, will see later if required to look-up the prj structure.

File: app-config.yaml (hunk 2 above)
Comment (fosterm-mw) [reply app-config.yaml:87]: @SANTOSH-MWP we have the grafana url: https://monitoring.px-dev.mavenwave.dev/

Comment (fosterm-mw) [reply app-config.yaml:87]: We may want to redo this PR since a lot of the code has been updated

**Comment by fosterm-mw** [comment]
I will close this PR because these changes are out of date, we will need to rebase and resubmit this PR

**Pull request closed without merging**